pkgs/applications/networking/instant-messengers/discord/disable-breaking-updates.py

```diff
@@ -25,7 +26,11 @@
 
 if os.path.exists(settings_path):
     with settings_path.open(encoding="utf-8") as settings_file:
-        settings = json.load(settings_file)
+        try:
+            settings = json.load(settings_file)
+        except json.JSONDecodeError:
+            print("[Nix] settings.json is malformed, letting Discord fix itself")
```

Comment:
```suggestion
            print(f"[Nix] {settings_file} is malformed; remove it to let Discord fix itself")
```
Perhaps tell the user where the file is and what to do?

Reply:
the user doesn't need to do anything

Reply:
Oh I see, my mistake.  I assumed this was a wrapper script and that the `sys.exit(0)` meant it wouldn't proceed to launch discord itself.